Makefile

```diff
@@ -1,6 +1,7 @@
 CC=gcc -g -Wall
 OFLAGS=-lm
-OBJ=src/vic.o src/vic-serial.o src/vic-funcs.o src/vic-exec.o src/vic-var.o
+OBJ=src/vic.o src/vic-serial.o src/vic-funcs.o src/vic-exec.o src/vic-var.o \
```

Comment:
@mrshu I have not seen many makefiles yet, so I worry I don't know how to do it. But if we are going to improve makefiles, I suggest also to look at tests/Makefile, when running all-tests, I see 'All passed' even when some don't pass.